app/assets/javascripts/articles.js

```diff
@@ -1,13 +1,21 @@
 App.articleQueue.onIncoming = function(data) {
-  var alertDiv = $('#floating-alert');
+  var alertDiv = document.querySelectorAll('#floating-alert');
```

Comment:
since `#floating-alert` is an ID it will only ever be matched one element, so you might want to use `document.querySelector('#floating-alert')`. Then you can avoid some headache below.